doc/man7/EVP_PKEY-DSA.pod

```diff
@@ -0,0 +1,65 @@
+=pod
```

Comment:
Good idea - but maybe a bit premature.
This is going to require quite a few additions AFTER the DSA/DH keygen PR's goes in.

Reply:
Yes?

So if this goes in first, add the new stuff in the appropriate pages...

Reply:
Which then delays us getting the PR's in.

Reply:
Fair enough. However, I actually don't expect this to go in before alpha1. Surely, you noticed WIP...

Reply:
Today it is WIP :)

Reply:
So...  how about actually reviewing?  If you look, I'm sure you'll see missing pieces that could be fleshed out.  There's even a `[TBA]` that I need help replacing.

Reply:
Yep will do, after the FFC code goes in.
Patience grasshopper :)

Reply:
> Patience grasshopper :)

Right back at ya ;-)

Reply:
I have had a go at updating the DSA/DH file now that the keygen for this is in.

Reply:
it was a big bigger than the [TBA] bit :)